tests/development/message_batches/create_message_batches/test_too_large.py

```diff
@@ -0,0 +1,77 @@
+import requests
+import pytest
+import uuid
+import json
+from lib import Assertions, Generators
+from lib.fixtures import *  # NOSONAR
+from lib.constants.message_batches_paths import MESSAGE_BATCHES_ENDPOINT
+
+MESSAGE_LIMIT = 45000
+CONTENT_TYPE = "application/json"
+
+
+@pytest.mark.devtest
+def test_too_many_messages(nhsd_apim_proxy_url, bearer_token_internal_dev):
+    """
+    .. include:: ../../partials/too_large/test_too_many_messages.rst
+    """
+
+    data = Generators.generate_valid_create_message_batch_body("dev")
+    messages = []
+    data["data"]["attributes"]["messages"] = messages
+
+    for i in range(MESSAGE_LIMIT+1):
+        messages.append({
+            "messageReference": str(uuid.uuid1()),
+            "recipient": {
+                "nhsNumber": "x"
+                }
+            })
+    # make sure it's less than 6MB to be a fair test
+    assert len(json.dumps(data)) < 6000000
+    resp = requests.post(f"{nhsd_apim_proxy_url}{MESSAGE_BATCHES_ENDPOINT}", headers={
+            "Authorization": bearer_token_internal_dev.value,
+            "Accept": CONTENT_TYPE,
+            "Content-Type": CONTENT_TYPE
+        },
+        json=data
+    )
+
+    Assertions.assert_error_with_optional_correlation_id(resp, 413, None, None)
+    assert len(resp.json()["errors"]) == 1
```

Comment:
I think it's worth checking the content of the error. The code CM_TOO_FEW_ITEMS at least.

Reply:
As above

Reply:
Looks good.